website_cookie_notice/views/website.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<template id="loader" name="Cookies script loader">
+    <script type="text/javascript"
+        src="/website_cookie_notice/static/src/js/accept_cookies.js">
+    </script>
+</template>
+
+<template id="message" name="Cookies notice">
+    <div t-if="request and
+        not request.httpsession.get('accepted_cookies')"
+        id="website_cookie_notice"
+        class="container-fluid bg-warning cc-cookies">
+        <div class="container">
+            <div class="row">
+                <div class="col-xs-10">We use cookies in this website. Read about them in our <a href="/page/privacy">privacy policy</a>. To disable them, configure your browser properly. If you keep using this website, you are accepting those.</div>
+                <div class="col-xs-2 text-center">
```

Comment:
Maybe add an offset here to keep the button and the text from smashing together? `col-xs-1 col-xs-offset-1`?